components/google-cloud/google_cloud_pipeline_components/remote/aiplatform/Dockerfile

```diff
@@ -20,7 +20,6 @@ WORKDIR /root
 # Upgrade pip to latest
 RUN pip3 install --upgrade pip
 
-# TODO() change from using PR commit id to release 
```

Comment:
should this still be neede?
And how do we tell the source code version for a specific GCPC SDK version?

Reply:
This is covered in the release instructions however I agree it is a good point to have it as a comment as well. I will add it back.

>And how do we tell the source code version

The `google_cloud_pipeline_components.__version__` would give the version of the pipeline being used in this container. 
users can inspect a container with 

```
docker run --entrypoint /bin/bash  gcr.io/managed-pipeline-test/gcp-launcher:v7  -c  "python3 -c 'import google_cloud_pipeline_components as gcpc; print(gcpc.__version__)'"
```